core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -210,7 +215,10 @@ object ConsumerPerformance {
    
     val useNewConsumer = options.has(useNewConsumerOpt)
     
-    val props = new Properties
+    val props = if (options.has(consumerConfigOpt))
+      Utils.loadProps(options.valueOf(consumerConfigOpt))
```

Comment:
Should these be added before or after the command line options? If we load them before, the default values for options would override custom settings from the `consumer.config` file even if the command line parameters were not specified.